CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing
+
+Contributions are welcome from anyone.
+
+To report bugs or issues, open [new issues](https://github.com/brndnmtthws/conky/issues/new).
+To submit code changes, open [new pull requests](https://github.com/brndnmtthws/conky/compare).
+
+## Guidelines
+
+Patches submitted in issues, email, or elsewhere may be ignored. When submitting PRs, please:
+
+- Describe the changes, why they were necessary, etc
+- Describe how the changes affect existing behaviour
+- Describe how you tested and validated your changes
+- Include unit tests when appropriate
+- Include any relevant screenshots/evidence demonstrating that the changes work and have been tested
+- Any new source files should include a GPLv3 license header
+- Any contributed code must be GPLv3 licensed
+- Always leave the code better than you found it
+- PRs with failed checks may be ignored or closed; please make sure
+  the build and checks pass if possible (and notify someone when the build
+  system is not working)
+
+## Coding Style
+
+Code should be formatted using `clang-format`. By configuring Conky with `cmake -DCHECK_CODE_QUALITY=ON`, you will be able to run `make clang-format` to automatically format code.
+
+If code in your PR is not formatted according to [`.clang-format`](.clang-format), the checks will not pass.
```

Comment:
(Imho). Don't break up lines. It looks slightly bad on `README.md`.  Keep them together.

Reply:
Noted.